common/certs/certs.go

```diff
@@ -24,6 +24,35 @@ import (
 // - https://github.com/grpc/grpc-go/blob/master/examples/features/encryption/mTLS
 // - and https://gist.github.com/annanay25/43e3846e21b30818d8dcd5f9987e852d.
 
+// CreateRootCA creates a new root certificate authority (CA) certificate and private key pair with the serial number and common name provided.
+// Only the cert is written into destDir in the PEM format. Being a CA, the certificate and private key returned can be used to sign other certificates.
+func CreateRootCA(commonName string, serialNo *big.Int, destDir string) (rootCert *x509.Certificate, rootKey *ecdsa.PrivateKey, err error) {
```

Comment:
So, you correctly move it now to the top, BUT, I think the tests should also use the same order then! :)